src/modules/shared/components/MediaCardList/MediaCardList.tsx

```diff
@@ -12,7 +12,7 @@ import { MEDIA_CARD_LIST_GRID_BP_COLS } from './MediaCardList.const';
 import styles from './MediaCardList.module.scss';
 import { MediaCardListProps } from './MediaCardList.types';
 
-const MediaCardList: FC<MediaCardListProps> = ({
+const MediaCardList: FC<MediaCardListProps & DefaultComponentProps> = ({
```

Comment:
In de rest van de client wordt de uitbreiding met `DefaultComponenProps` in de `.types` files gedaan. Ik zou dat voor consistentie hier ook zo doen. Wel approved